packages/core/src/eventUtils.ts

```diff
@@ -57,7 +57,7 @@ export function createErrorPlatformEvent(
   id: string,
   data?: unknown
 ): ErrorPlatformEvent {
-  return { type: `error.platform.${id}`, data };
+  return { type: `xstate.error.platform.${id}`, data };
```

Comment:
To accompany the other proposed change, I propose to rename this to `xstate.error.actor.INVOKE_ID`.